capstone/src/test/resources/com/google/univiz/scorecard/ScorecardTest.java

```diff
@@ -0,0 +1,37 @@
+package com.google.univiz.scorecard;
```

Comment:
This class should be in `src/test/java/com/google/univiz/scorecard`. As it is, your tests aren't getting invoked by Maven. Can you move it and then get the test to pass?